tests/h/jinja2_extensions/svg_icon_test.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+import pytest
+from markupsafe import Markup
+
+from h.jinja_extensions import svg_icon
+
+
+class TestSVGIcon:
+    def test_it_removes_title(self, icon_file):
+        icon_file.write(
+            '<svg xmlns="http://www.w3.org/2000/svg"><title>foo</title></svg>'
+        )
+
+        assert svg_icon("test_icon_name") == Markup(
+            '<svg xmlns="http://www.w3.org/2000/svg" class="svg-icon" />'
+        )
+
+    def test_it_strips_default_xml_namespace(self, icon_file):
+        icon_file.write('<svg xmlns="http://www.w3.org/2000/svg"></svg>')
+
+        assert svg_icon("test_icon_name") == Markup(
+            '<svg xmlns="http://www.w3.org/2000/svg" class="svg-icon" />'
+        )
```

Comment:
This appears to prove nothing of the sort with regards to the test name, but it's what we had before.